VOTING_MEMBERS.md

```diff
@@ -76,6 +76,8 @@ The Voting Members, in alphabetic order by their GitHub handles, are:
 
 [@Kanahiro](https://github.com/Kanahiro) (Mierune)
 
+[@keith](https://github.com/keith) (Lyft)
```

Comment:
```suggestion
[@keith](https://github.com/keith) (Modular)
```

I actually switched jobs earlier  this year, which means I am no longer a direct user of maplibre, I'm not sure how that affects my eligibility here, but I definitely understand if it does

Reply:
It doesn't. However, if you won't have any involvement with MapLibre, you can also choose to not accept the nomination. Up to you. :-)

Reply:
Thanks for the feedback @keith! I will close this Pull Request.